source/cursorManager.py

```diff
@@ -26,9 +26,13 @@
 import braille
 import vision
 import controlTypes
-from inputCore import SCRCAT_BROWSEMODE

```

Comment:
I am a bit concerned by the use of SCRCAT_BROWSEMODE for the scripts to got to next/previous paragraph, since they could be used in plain text not read in browse mode.
This concern probably applies to other existing scripts however and it may be out of the scope of this PR.

Reply:
Yes maybe a new category should be created for document navigation, like the settings panel

Reply:
@CyrilleB79 / @rob-aph what do you think of this suggestion to create a new script category?

Reply:
Yes, I fully agree with a new category.
In the future:
* other scripts can be added to this category, such as a future sentence navigation scripts (#8518)
* it should be thought if some options/scripts from other categories/panels should land in this category

Reply:
I guess I'm not exactly sure what we are talking about. If we are talking about scripts listed under Input Gestures in Preferences, none of the scripts in cursorManager.py are listed there. So, it's unclear to me what the line scriptCategory=SCRCAT_BROWSEMODE even does with regard to the CursorManager class. None of these commands can be reassigned.
Also, CursorManager had basic paragraph navigation long before I came along, so how does this relate to my improved paragraph navigation pr?

Reply:
> If we are talking about scripts listed under Input Gestures in Preferences, none of the scripts in cursorManager.py are listed there. So, it's unclear to me what the line scriptCategory=SCRCAT_BROWSEMODE even does with regard to the CursorManager class. None of these commands can be reassigned.

Yes, there are 3 scripts that can be reassigned: find, find next and find previous. The other ones have no description and thus cannot be reassigned. This is intended because they match standard windows shortcuts to move the cursor or the selection and are just here to vocalize the result of this action.

> Also, CursorManager had basic paragraph navigation long before I came along, so how does this relate to my improved paragraph navigation pr?

Yes, as written before, maybe this suggestion is a bit off-topic for this PR since it does have effect on paragraph navigation scripts which cannot be reassigned.

Reply:
This has not been resolved yet

Reply:
See resolution in https://github.com/nvaccess/nvda/pull/13798#discussion_r991831628